packages/react/transform/__test__/fixture.spec.js

```diff
@@ -938,12 +939,14 @@ describe('dynamic import', () => {
     );
 
     expect(result.code).toMatchInlineSnapshot(`
-      "import "@lynx-js/react/experimental/lazy/import";
+      "import "@lynx-js/react/experimental/lazy/lynx";
+      import "@lynx-js/react/experimental/lazy/import";
```

Comment:
I think this test case should be split and combined correctly:

- Only `lazy/lynx` is generated
- Only `lazy/import` is generated
- Both `lazy/lynx` and `lazy/import` are generated